Shopify/cli
# Add silence flag to the theme dev commands

### WHY are these changes introduced?
Closes: https://github.com/Shopify/cli/issues/4714
I saw a slack [post](https://shopifypartners.slack.com/archives/CKQTKDZV2/p1732530699498259l) talking about request logs (Ex: `• 12:57:27 Request »    GET 200 /collections/all 51`) are too noisy and they wish they could be disabled.
Fixes #0000 

### WHAT is this pull request doing?

Add a `--silence flag to the theme dev command to disable request logs`

Before:
![image](https://github.com/user-attachments/assets/9661575d-bfa4-4fa3-96a7-f7ea13206384)

After:
![image](https://github.com/user-attachments/assets/f6806ac3-d94c-40ca-82bb-a0ae1d641798)

### How to test your changes?

Pull down the branch
Build the branch
Run `theme dev --silence` and click around in your theme links like collections, products, etc.
You shouldn't see any request logs in the terminal.

### Post-release steps

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist

- [] I've considered possible [documentation](https://shopify.dev) changes
